pontoon/base/templatetags/helpers.py

```diff
@@ -1,8 +1,9 @@
 import cgi
 import datetime
 import json
-import urllib
-import urlparse
+
+from six import text_type
+from six.moves.urllib import parse as urlparse
```

Comment:
@adngdb Are there really such things in `urlparse` that are not supported by [`six.moves.urllib.parse`](https://pythonhosted.org/six/#module-six.moves.urllib.parse)? Wouldn't people notice they break if they use them anyways?

Anyhow, if you decide to use `six_parse`, then we should do this across the entire project, not just in this file.